src/components/DisplayNames/DisplayNamesPropTypes.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    // The full title of the DisplayNames component (not split up)
+    fullTitle: PropTypes.string,
+
+    // Array of objects that map display names to their corresponding tooltip
+    displayNamesToTooltips: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
NAB, kind of a weird name since it sounds more like a utility method than a list. I'd change this to `displayNamesWithTooltipsList` or something.